components/brave_rewards/resources/tip/components/siteBanner.tsx

```diff
@@ -215,6 +217,7 @@ class Banner extends React.Component<Props, State> {
     const checkmark = utils.isPublisherConnectedOrVerified(publisher.status)
     const bannerType = this.props.monthly ? 'monthly' : 'one-time'
     let logo = publisher.logo
+    const currentBalance = total && total.toFixed(1) || 0
```

Comment:
to fixed return string, so we could just say `"0.0"` and this way we don't need string at the bottom

Reply:
Right, but if `total` is invalid the evaluation will not continue to `toFixed(1)`. I've updated this line to: `const currentBalance = total && total.toFixed(1) && total.toFixed(1).toString() || '0.0'
`

Reply:
toFixed returns string so you don't need toString

Reply:
so it can be like this `const currentBalance = total && total.toFixed(1) || '0.0'

Reply:
done